X11.h defines a macro called "Success", which is 0. This breaks an enum
that is generated in Chromium that has a member "Success", so undefine
it.
--- a/chrome/browser/ui/libgtkui/select_file_dialog_impl_kde.cc.orig	2020-04-27 18:42:15.457297544 +0200
+++ b/chrome/browser/ui/libgtkui/select_file_dialog_impl_kde.cc	2020-04-27 18:42:21.760454304 +0200
@@ -24,7 +24,9 @@
 #include "base/task/post_task.h"
 #include "base/task/task_traits.h"
 #include "base/threading/thread_restrictions.h"
+#undef Success
 #include "chrome/browser/ui/libgtkui/select_file_dialog_impl.h"
+#define Success 0
 #include "content/public/browser/browser_thread.h"
 #include "ui/aura/window_tree_host.h"
 #include "ui/base/l10n/l10n_util.h"
